<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grp_steps">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Manual Update Step Motors</string>
    </property>
    <widget class="QLabel" name="lbl_stepy">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y Step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_stepX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stepX1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X Step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_stepY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xPositive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xNegative">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yPositive">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yNegative">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="keyboardCheckBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Keyboard</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_msg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Messages</string>
    </property>
    <widget class="QTextBrowser" name="message_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_connection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QLabel" name="lbl_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_conn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>94</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_disconn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>94</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_ip">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_port">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>3965</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_system">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>System</string>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>74</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioManuel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioAutomatic">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic Mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_frame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Camera Frames</string>
    </property>
    <widget class="QLabel" name="lbl_frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_route">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>490</width>
      <height>360</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Route - Centimeter</string>
    </property>
    <widget class="QGraphicsView" name="vp_route">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>450</width>
       <height>310</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rulerH">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0   1   2   3   4    5   6   7   8   9  10  11  12  13 14 15  16 17  18 19  20 21 22 23 24 25 26 27</string>
     </property>
    </widget>
    <widget class="QLabel" name="rulerV">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>40</y>
       <width>20</width>
       <height>311</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>231</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhMultiLine</set>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="gtuLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Current Step Coordinates</string>
    </property>
    <widget class="QLabel" name="lbl_xcoord">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="yCoordinate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_ycoord">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Y :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="xCoordinate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCSE396">
    <property name="title">
     <string>CSE396</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCSE396"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
